### DISCUSSION TITLE
Ubuntu 22.04.4 LTS

### DISCUSSION THREAD

**Discussion by Bouli911** [AdguardTeam/AdGuardHome#7007] [Q&A]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Linux, ARM64

### Installation

GitHub releases or script from README

### Setup

On a router, DHCP is handled by AdGuard Home

### AdGuard Home version

Version: v0.107.48

### Action

speedtest 

all my computer using OS ubuntu Server: 

Distributor ID: Ubuntu
Description:    Ubuntu 22.04.4 LTS
Release:        22.04
Codename:       jammy

when i do speedtest command line "speedtest-cli" i experience 12 mbps download and 12 mbps upload ..

if i change my dns to 1.1.1.1 and  1.0.0.1  i do 2717mbps down and 2382 upload..

i have try to disable systemd-resolved.service completly with no result..

My adguard running on a raspberypi5 and ubuntu 22 .04.4 lts .

on a windows computer if i do a speed test with adguardhome i have 3200 mbps down and 2900 upload

error on  systemd-resolved.service status ( sudo systemctl status systemd-resolved.service) :
Using degraded feature set UDP instead of UDP+EDNS0 for DNS server 192.168.80.16.
Grace period over, resuming full feature set (UDP+EDNS0) for DNS server 192.168.80.16.

### Expected result

doing same speed as using dns 1.1.1.1 on Ubuntu 22.04.4 LTS

### Actual result

experience 12 mbps download and 12 mbps upload

**Comment by agneevX** [comment]
> when i do speedtest command line "speedtest-cli" i experience 12 mbps download and 12 mbps upload ..
> 
> if i change my dns to 1.1.1.1 and 1.0.0.1 i do 2717mbps down and 2382 upload..

Speedtest.net uses HTTP pings to nearby servers to determine the one with the lowest latency. If you are using a DNS server that is located far away from you, it might use it to guess your location since it does not have access GPS on the CLI. 1.1.1.1 is obviously closer to you since Cloudflare has a lot of edge locations.

**Comment by Bouli911** [comment]
I use same dns server on both config and same server for speed test ..  …i use systemd-resolved.service  and netplan maybe its netplan … don’t know
